File: Ejemplo.cpp
#include <iostream>
using namespace std;

// Función para sumar dos números
int sumar(int a, int b) {
    return a + b;
}

int main() {
    string nombre;
    int num1, num2;

    cout << "Ingrese su nombre: ";
    cin >> nombre;

    cout << "Ingrese dos numeros: ";
    cin >> num1 >> num2;

    int resultado = sumar(num1, num2);

    cout << "Hola " << nombre << endl;
    cout << "La suma es: " << resultado << endl;

    return 0;
}

File: Ejemplo.cs
using System;

class Program
{
    static int Sumar(int a, int b)
    {
        return a + b;
    }

    static void Main()
    {
        Console.Write("Ingrese su nombre: ");
        string nombre = Console.ReadLine();

        Console.Write("Ingrese dos numeros: ");
        int num1 = int.Parse(Console.ReadLine());
        int num2 = int.Parse(Console.ReadLine());

        int resultado = Sumar(num1, num2);

        Console.WriteLine($"Hola {nombre}");
        Console.WriteLine($"La suma es: {resultado}");
    }
}
